code/ui/Hud.cs

```diff
@@ -33,20 +35,54 @@ public static void OnHotReloaded()
                 Local.Hud?.Delete();
 
                 Hud hud = new();
+
+                if (Local.Client.Pawn is TTTPlayer player && player.LifeState == LifeState.Alive)
+                {
+                    hud.AliveHudPanel.Enabled = true;
+                }
+            }
+        }
+
+        [Event("tttreborn.player.spawned")]
+        private void OnPlayerSpawned(TTTPlayer player)
+        {
+            if (player != Local.Client.Pawn)
+            {
+                return;
             }
+
+            Current.AliveHudPanel.Enabled = true;
         }
 
-        public class GeneralHud : TTTPanel
+        [Event("tttreborn.player.died")]
+        private void OnPlayerDied(TTTPlayer player)
         {
-            public GeneralHud(Panel parent)
+            if (player != Local.Client.Pawn)
+            {
+                return;
+            }
+
+            Current.AliveHudPanel.Enabled = false;
+        }
+
+        public class GeneralHud : Panel
+        {
+            public GeneralHud(Sandbox.UI.Panel parent) : base(parent)
             {
                 Parent = parent;
 
-                Parent.AddChild<PlayerInfo>();
+                Parent.AddChild<RadarDisplay>();
```

Comment:
I'd have kept it separated and in the radar item file to support code-separation.

Reply:
https://github.com/TTTReborn/tttreborn/issues/219